libraries/lib-wave-track/AudioSegmentSampleView.h

```diff
@@ -9,19 +19,47 @@
 
 using BlockSampleView = std::shared_ptr<std::vector<float>>;
 
-// An audio segment is either a whole clip or the silence between clips.
-class WAVE_TRACK_API AudioSegmentSampleView
+//! An audio segment is either a whole clip or the silence between clips.
+class WAVE_TRACK_API AudioSegmentSampleView final
 {
 public:
+   /*
+    * @brief Constructs an `AudioSegmentSampleView` from `BlockSampleView`s.
```

Comment:
The grave accents are markdown that make text appear in typewriter font.  I often use it in `@pre` and `@post` when a pre/post-condition can be written formally in C++.

It's less important here -- Doxygen will recognize the class name and format it as an html link.  The grave accents cause the link to have typewriter font too.

"BlockSampleViews" however will fail to link because it's not the exact type alias name.